reports.py

```diff
@@ -363,10 +363,11 @@ def deleted_prods(self):
         self.publish_report("deletedprods-page-title", text)
 
     def most_used_templates(self):
-        query = """SELECT tl_title, COUNT(*)
+        query = """SELECT lt_title, COUNT(*)
                     FROM templatelinks
-                    WHERE tl_namespace = 10
-                    GROUP BY tl_title
+                    JOIN linktarget ON tl_target_id = lt_id
+                    WHERE lt_namespace = 10
+                    GROUP BY lt_title
                     ORDER BY COUNT(*) DESC
                     LIMIT 3000"""
```

Comment:
Both `lt_*` and `tl_*` being used is surely never going to cause confusion :')

Looking [at phab](https://phabricator.wikimedia.org/T299417#7818483) it seems correct — just testing this (pretty slow..) query and then will review ^^

Reply:
wfm, `Pagetype` being the top returned template :)

Reply:
> Both `lt_*` and `tl_*` being used is surely never going to cause confusion :')

Yup! Ladsgroup himself got confused at https://phabricator.wikimedia.org/T299417#7814637 with `lt_target_id = lt_id`, teehee!

Thanks for the swift review